modules/export_pipeline/export_to_aws.tf

```diff
@@ -88,7 +88,7 @@ resource "google_cloudfunctions2_function" "export_to_aws_function" {
 
   service_config {
     available_memory      = "4Gi"
-    timeout_seconds       = 540
+    timeout_seconds       = 3600
```

Comment:
540 is the maximum
https://cloud.google.com/functions/quotas#time_limits

hopefully it doesn't actually take an hour to upload a file to AWS!
multi-part uploads should improve upload time:
https://docs.aws.amazon.com/AmazonS3/latest/userguide/mpuoverview.html
so I'd make sure you're using them

Reply:
This is an HTTP function, which has a 60 min timeout. FWIW uploading ~500 files took 15 min so we won't actually hit this max timeout: https://cloud.google.com/functions/docs/configuring/timeout#:~:text=The%20maximum%20timeout%20duration%20is,immediately%20returned%20to%20the%20caller.